opt_out_http_api/api_methods.py

```diff
@@ -45,12 +54,26 @@ def response(self, request, status_code=200, status_reason="OK", **data):
         })
         return json.dumps(data)
 
-# GET Method
+# Error Handling
+
+    @app.handle_errors(OptOutNotFound)
+    def opt_out_not_found(self, request, failure):
+        return self.response(
+            request, status_code=404, status_reason="Opt out not found.")
+
+    @app.handle_errors(OptOutAlreadyExists)
+    def opt_out_already_exists(self, request, failure):
+        return self.response(
+            request, status_code=409, status_reason="opt out already exists.")
```

Comment:
We should probably capitalize the "o" in "opt out" for consistency with the error message "Opt out not found".